Source/WebCore/rendering/svg/SVGInlineTextBox.h

```diff
@@ -23,14 +24,21 @@
 
 #include "LegacyInlineTextBox.h"
 #include "LegacyRenderSVGResource.h"
+#include "RenderSVGResourcePaintServer.h"
 #include "SVGTextFragment.h"
 
 namespace WebCore {
 
 class LegacyRenderSVGResource;
 class RenderSVGInlineText;
+class SVGPaintServerHandling;
 class SVGRootInlineBox;
 
+#if ENABLE(LAYER_BASED_SVG_ENGINE)
+class RenderSVGResourcePaintServer;
```

Comment:
I suggest either forward referencing or including, not both.

Reply:
Fixed.